### PR TITLE
March 11th polls

### PR DESCRIPTION
**Pull request by MakerJanSky**

### CODE REVIEW

File: governance/polls/Adjust SparkLend D3M Parameters - March 11, 2024.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Adjust SparkLend D3M Parameters - March 11, 2024
+summary: Signal your support or opposition to increasing the SparkLend DAI Direct Deposit Module (D3M) debt ceiling to 2.5 billion DAI.
```
Comment (0xldr) [inline governance/polls/Adjust SparkLend D3M Parameters - March 11, 2024.md:3]:
```suggestion
summary: Signal your support or opposition to increasing the SparkLend DAI Direct Deposit Module (D3M) Maximum Debt Ceiling to 2.5 billion DAI.
```

File: governance/polls/Adjust SparkLend D3M Parameters - March 11, 2024.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to making the following parameter changes to the $protocol_name DAI Direct Deposit Module (D3M):
+* Increase the SparkLend D3M [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 1.25 billion DAI to **2.5 billion DAI**.
```
Comment (0xldr) [inline governance/polls/Adjust SparkLend D3M Parameters - March 11, 2024.md:30]: add difference and use definition from the MIP

```suggestion
* Increase the SparkLend D3M [Maximum Debt Ceiling](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) by 1.25 billion DAI from 1.25 billion DAI to **2.5 billion DAI**.
```

File: governance/polls/SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 11, 2024.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+* Increase the WBTC Supply Cap `max` by 1,000 WBTC, from 5,000 to **6,000 WBTC**.
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 11, 2024.md:30]: nit - consider adding a definition for the `max`

**Review by 0xldr — CHANGES_REQUESTED** [review]
please see comments

File: governance/polls/Adjust SparkLend D3M Parameters - March 11, 2024.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of SparkLend. The community can vote in this poll to express support or opposition to using the SparkLend Direct Deposit DAI Module (D3M) with the listed parameters. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, March 11 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to making the following parameter changes to the $protocol_name DAI Direct Deposit Module (D3M):
```
Comment (0xldr) [inline governance/polls/Adjust SparkLend D3M Parameters - March 11, 2024.md:29]:
```suggestion
The community can vote in this poll to express support or opposition to making the following parameter changes to the SparkLend DAI Direct Deposit Module (D3M):
```

File: governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP23) - March 11, 2024.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-03-11T16:00:00
+end_date: 2024-03-25DDT16:00:00
```
Comment (0xldr) [inline governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP23) - March 11, 2024.md:25]:
```suggestion
end_date: 2024-03-25T16:00:00
```

**Review by 0xldr — CHANGES_REQUESTED** [review]
One change then good to merge

File: governance/polls/SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 11, 2024.md (hunk 3 above)
Comment (MakerJanSky) [reply governance/polls/SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 11, 2024.md:30]: In future polls, we'll include some definitions or documentation.

File: governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP23) - March 11, 2024.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP102c2-SP23 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This subproposal seeks to make various updates to the Alignment Artifacts to prepare for Phase 1 of Endgame.*
+
+Please review the links below to inform your position on this proposal before voting.
+* Canonical Proposal Version [MIP104](https://github.com/makerdao/mips/blob/6e7469bebae64210ac44420eb7772bc610a2cb2c/MIP104/MIP104.md), [MIP113](https://github.com/makerdao/mips/blob/6e7469bebae64210ac44420eb7772bc610a2cb2c/MIP113/MIP113.md)
```
Comment (0xldr) [inline governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP23) - March 11, 2024.md:45]:
```suggestion
* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/ec87f8bd4dfe7307698e7dc87da19014e466eb1d/MIP102/MIP102c2-Subproposals/MIP102c2-SP23.md)
```

File: governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP23) - March 11, 2024.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+* Latest Proposal Version [MIP104](https://mips.makerdao.com/mips/details/MIP104), [MIP113](https://mips.makerdao.com/mips/details/MIP113)
```
Comment (0xldr) [inline governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP23) - March 11, 2024.md:46]:
```suggestion
* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP102c2SP23)
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

**Review by 0xldr — APPROVED** [review]

**Pull request merged**